src/pages/patch/PatchRestartModal.tsx

```diff
@@ -28,14 +28,19 @@ interface PatchModalProps {
   visible: boolean;
   onOk: () => void;
   onCancel: () => void;
+  patchId?: string;
+  refetchQueries: string[];
 }
 export const PatchRestartModal: React.FC<PatchModalProps> = ({
   visible,
   onOk,
   onCancel,
+  patchId: patchIdFromProps,
+  refetchQueries,
 }) => {
   const dispatchBanner = useBannerDispatchContext();
   const { id } = useParams<{ id: string }>();
+  const patchId = patchIdFromProps || id;
```

Comment:
this is a great place to use the nullish coalescing operator

Reply:
done!